### ISSUE TITLE
TwilioVideo.connect can return nil according to docs but that is not possible

### ISSUE THREAD

**Issue by andschdk** [twilio/twilio-video-ios#14]
I believe the `TwilioVideo.connect` method could return `nil` once, but that is not the case anymore and the documentation needs an update.

The case where `nil` was returned previously is now going to the delegate I assume?

From the public header file:
```objc
/**
 *  @brief Connect to a Room with options and a delegate.
 *
 *  @param options  The connect options.
 *  @param delegate A delegate conforming to `TVIRoomDelegate` for handling Room related events.
 *
 *  @return An instance of `TVIRoom` if successful and `nil` otherwise.
 */
+ (nonnull TVIRoom *)connectWithOptions:(nonnull TVIConnectOptions *)options
                               delegate:(nullable id<TVIRoomDelegate>)delegate;
```

#### Video iOS SDK

[2.0.0 via CocoaPods]

**Comment by ceaglest** [comment]
Hey @andschdk,

Thanks for filing an issue. It appears that the documentation for `@return` is incorrect. We will fix this in an upcoming release.

Regards,
Chris

**Comment by paynerc** [comment]
The documentation was been corrected in the `2.2.0` release which was released yesterday.

Thanks again for reporting this discrepancy!

Ryan
